filebeat/module/pensando/dfw/test/test.log-expected.json

```diff
@@ -0,0 +1,113 @@
+[
+    {
+        "@timestamp": "2020-04-22T17:24:31.105Z",
+        "destination.address": "172.31.10.2",
+        "destination.port": "22",
```

Comment:
I think this and other `*port` values need to be converted to integers in the pipeline

Reply:
Is this done using the convert processor at the end of the pipeline file?  Good idea, just not sure how to proceed. Thanks

Reply:
I would have thought this was done in the fields.yml file but apparently not.

Reply:
As you said you can do it using the [convert processor](https://www.elastic.co/guide/en/elasticsearch/reference/current/convert-processor.html), for example:

```yaml
- set:
    field: destination.port
    value: '{{pensando.dfw.destination_port}}'
    ignore_failure: true
```

Could be set as:

```yaml
- convert:
    field: pensando.dfw.destination_port
    target_field: destination.port
    type: integer
    ignore_failure: true
    ignore_missing: true
```

Reply:
Fixed this for both ports and src/dst ip fields : 849a1a0aaf220af7e1617f3ef45c0bd10dc588b8